lua/weapons/laserpointer/cl_init.lua

```diff
@@ -16,35 +18,59 @@ function SWEP:Setup(ply)
 			self.Attach = attachmentIndex
 		end
 	end
+
 	if ply:IsValid() then
 		local attachmentIndex = ply:LookupAttachment("anim_attachment_RH")
 		if ply:GetAttachment(attachmentIndex) then
-			self.WM = ply
 			self.WAttach = attachmentIndex
 		end
 	end
 end
+
 function SWEP:Initialize()
 	self:Setup(self:GetOwner())
 end
+
 function SWEP:Deploy(ply)
 	self:Setup(self:GetOwner())
 end
 
+function SWEP:PrimaryAttack()
```

Comment:
This needs to be networked so other clients see its correct state. DTVars is the best way but NWVars works too.

Reply:
I was experimenting to see if the network vars could be phased out, but it's pretty wonky without them. I'll just keep it simple and add them back